web/tests/functional/report_viewer_api/test_report_filter.py

```diff
@@ -382,7 +382,7 @@ def test_detection_date_filters(self):
     def test_fix_date_filters(self):
         """ Filter by fix dates. """
         report_filter = ReportFilter(
-            detectionStatus=[DetectionStatus.RESOLVED])
+            detectionStatus=[DetectionStatus.OFF])
```

Comment:
Can you provide more context for this change?

Reply:
Similar issue than the previous one. It comes from the "disable vs. hide" semantic change. The "OFF" status means that the `unix.Malloc` checker has been disabled and that's the reason for the report being resolved. Earlier there was a `unix.MismatchedDeallocator` checker report which simply became resolved without the checker being disabled. I agree if it seems a little fuzzy. If you need more precise elaboration about the exact reasons, then I could take some more effort in its inspection once... maybe sometime... in the future... :)